.github/workflows/add-backport-label.yml

```diff
@@ -0,0 +1,14 @@
+name: Add Backport Label
+on:
+  pull_request_target:
+    branches: main
+    types: opened
+
+jobs:
+  add_labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: backport 2.x
```

Comment:
Shouldn't we follow a similar branch naming convention to [SDK repo](https://github.com/opensearch-project/opensearch-sdk-java/tree/1.x) and have a 1.x branch that's compatible to OpenSearch 2.x, and a main branch that's compatible to OS 3.x